### ISSUE TITLE
Togi - ukol3

### ISSUE THREAD

**Issue by Togi12** [Togi12/python-kurz-2022#3]
ahoj, tady je můj třetí ukol. Omlouvám se, že ho posílám takhle na poslední chvíli. Jediný co mi tam nesedí je ta fce round - jak se zaokrouhluje nahoru? Díky, Togi

https://github.com/Togi12/python-kurz-2022/blob/main/ukol-03.py

**Comment by pixelneo** [comment]
Ahoj, round je ok.
Jen ta funkce `cena` by mela tu cenu pocitat a vratit, tady ale neni volana.

**Comment by pixelneo** [comment]
jo, jinak nahoru se zaokrouhluje pomoci funkce `ceil` z modulu `math`. Na zacatek souboru napis `from math import ceil` a pak uz lze ceil pouzit: `ceil(1.23)`.

**Comment by Togi12** [comment]
jo, díky.. já stejně nějak nemůžu přijít na to, jak bych to udělala, tak počkám, až se to bude řešit na lekci v pondělí. Díky moc
